src/widget/button.rs

```diff
@@ -21,13 +21,13 @@ use crate::{
     UpdateCtx, Widget,
 };
 
-use crate::piet::{Color, FillRule, FontBuilder, Text, TextLayoutBuilder};
+use crate::piet::{Color, FontBuilder, Text, TextLayoutBuilder};
 use crate::{Piet, Point, RenderContext};
 
-const BUTTON_BG_COLOR: Color = Color::rgba32(0x40_40_48_ff);
-const BUTTON_HOVER_COLOR: Color = Color::rgba32(0x50_50_58_ff);
-const BUTTON_PRESSED_COLOR: Color = Color::rgba32(0x60_60_68_ff);
-const LABEL_TEXT_COLOR: Color = Color::rgba32(0xf0_f0_ea_ff);
+const BUTTON_BG_COLOR: Color = Color::rgba8(0x40, 0x40, 0x48, 0xff);
```

Comment:
I think if the alpha is 0xff, it should be omitted. OTOH this is going to go away soon :)